docs/reference/mount.md

```diff
@@ -88,6 +93,31 @@ The `umount` command is used to unmount a currently mounted lakeFS repository.
 everest umount <data_directory>
 ````
 
+### Diff Command (write-mode only)
+The `diff` command Show the diff between the source branch and the current mount directory. 
+If `<data_directory>` not specified, search for the mount directory in the current working directory and upwards based on `.everest` directory existence.
+Please note that the diffs are from the source branch state at the time of mounting and not the current state of the source branch.
+
+```bash
+everest diff <data_directory>
+
+#Example output:
+# - removed cats/persian/cute.jpg
+# ~ modified dogs/golden_retrievers/cute.jpg
+# + added birds/parrot/cute.jpg
+```
+
+### Commit Command (write-mode only)
+The `commit` command commits the changes made in the mounted directory to the original lakeFS branch.
+If `<data_directory>` not specified, search for the mount directory in the current working directory and upwards based on `.everest` directory existence.
+The new commit will be merged to the original branch with the `source-wins` strategy in case of conflicts.
+Writes during commit may be lost.
```

Comment:
I would put that as a warning.

Reply:
@itaiad200 
which sentence here?
Rhe one that warning about writes during a commit?